.github/workflows/release.yml

```diff
@@ -50,13 +50,38 @@ jobs:
       - name: Go Mod Tidy
         run: go mod tidy
 
-      - name: Go Build
+      - name: Compile for Linux amd64
         env:
           VERSION: ${{ needs.test.outputs.version }}
           CGO_ENABLED: 0
           GOOS: linux
           GOARCH: amd64
-        run: go build -ldflags "-s -w -X github.com/checkmarx/2ms/cmd.Version=$VERSION" -a -installsuffix cgo -o bin/2ms main.go
+        run: |
+          go build -ldflags "-s -w -X github.com/checkmarx/2ms/cmd.Version=$VERSION" -a -installsuffix cgo -o dist/2ms main.go
+          zip -j dist/linux-amd64-$VERSION.zip dist/2ms
+          rm dist/2ms
+
+      - name: Compile for MacOS amd64
+        env:
+          VERSION: ${{ needs.test.outputs.version }}
+          CGO_ENABLED: 0
+          GOOS: darwin
+          GOARCH: amd64
+        run: |
+          go build -ldflags "-s -w -X github.com/checkmarx/2ms/cmd.Version=$VERSION" -a -installsuffix cgo -o dist/2ms main.go
+          zip -j dist/macos-amd64-$VERSION.zip dist/2ms
```

Comment:
I have a problem with the name containing a version.

If you keep the name the same across versions, you can download the latest artifact with a link:
`https://github.com/Checkmarx/2ms/releases/latest/download/2ms`.

But now, we can't use a generic link, because the filename is always changing, and we need to first fetch the latest version name, then construct the filename to download.

Also, need to update the README now.